packages/graphql-server/vitest.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defaultExclude, defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    exclude: [...defaultExclude, 'bun_test'],
+    globals: true,
+    restoreMocks: true,
+    unstubEnvs: true,
+    onConsoleLog(_, type) {
+      // Do not show `console.error` messages
+      if (type === 'stderr') {
+        return false
+      }
```

Comment:
I'm unsure how useful this is.. it's just replicating the `spyOn(console, 'error')` that was here previously

https://github.com/honojs/middleware/blob/0f8d2f3d055e7873c22960db6a4a10e20bd75d91/packages/graphql-server/test/index.test.ts#L13-L14